workers/unity/Packages/io.improbable.gdk.mobile/Editor/iOSUtils.cs

```diff
@@ -179,14 +187,30 @@ public static void Launch(DeviceLaunchConfig deviceLaunchConfig, MobileLaunchCon
             }
         }
 
+        private static bool IsXCodeProjectForDevice()
+        {
+            /*
+             * Unity adds multiple libraries to the XCode project. One of them is libiPhone-lib.
+             * This library will be a dylib, if the Unity project was built for simulator, and otherwise it will be a
+             * static library.
+             */
+            return File.Exists(Path.Combine(XCodeProjectPath, "Library", "libiPhone-lib.a"));
+        }
+
         private static bool TryBuildXCodeProject(string developmentTeamId)
         {
+            var teamIdArgs = string.Empty;
+            if (!string.IsNullOrEmpty(developmentTeamId))
+            {
+                teamIdArgs = $"DEVELOPMENT_TEAM={developmentTeamId}";
+            }
```

Comment:
Is `developmentTeamId` an optional argument here? I.e. - can the build work without it?

Reply:
@jessicafalk

Reply:
if you build for simulator it will work without

Reply:
I think it can build without it for the simulator. Or so the error messages indicate